app/src/main/java/rkr/simplekeyboard/inputmethod/latin/settings/InputMethodSettingsFragment.java

```diff
@@ -29,6 +30,14 @@
 public abstract class InputMethodSettingsFragment extends PreferenceFragment
         implements InputMethodSettingsInterface {
     private final InputMethodSettingsImpl mSettings = new InputMethodSettingsImpl();
+    private final Handler handler =new Handler();
```

Comment:
What is being fixed here?

Reply:
To reproduce the bug in the old version do this:
1. Open Simple Keyboard settings
2. Open "Languages" (language chooser activity)
3. Add Armenian and navigate BACK.

Observed results:
You observe that the list of languages in the settings is not updated yet. It doesn't mention Armenian.

Expected results:
This SettingsFragment is expected to have the list updated immediately when returned to it.